### ISSUE TITLE
user interface - action items

### ISSUE THREAD

**Issue by johnabuck** [agilepro/cognoscenti#196]
The detailed view of Action Items has an item "Linked Topics".  I am unable to figure out how one can link a topic to an action item? If we have it, that feature seems nonobvious.

**Comment by agilepro** [comment]
you go to the topic, and link the action item to the topic.  

The full description is "Topics that link this Action Item"  but that is a bit wordy.

I ADDED some explanatory text in lightgrey, adn also in a tooltip:

![image](https://user-images.githubusercontent.com/5544889/34425105-f4437ed2-ebdd-11e7-9273-b9c2f04afe93.png)
